### PR TITLE
correct SP1 paths + some cleanup

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

**Comment by toabctl** [comment]
+1

**Comment by bmwiedemann** [comment]
added two more commits in this direction

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -510,15 +510,8 @@ function export_tftpboot_repos_dir()
...
-            tftpboot_suse12_dir=/srv/tftpboot/suse-12.0
-            tftpboot_repos12sp1_dir=$tftpboot_suse12sp1_dir/repos
-        else
-            tftpboot_suse_dir=/srv/tftpboot/suse-11.3/x86_64
-            tftpboot_suse12_dir=/srv/tftpboot/suse-12.0/x86_64
-            tftpboot_repos12sp1_dir=$tftpboot_suse12sp1_dir/x86_64/repos
-        fi
+        tftpboot_suse12_dir=/srv/tftpboot/suse-12.0/x86_64
+        tftpboot_repos12sp1_dir=$tftpboot_suse12sp1_dir/x86_64/repos
         tftpboot_repos_dir=$tftpboot_suse_dir/repos
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: `$tftpboot_suse_dir` isn't defined anymore.

Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: ah, then we should drop this line, too. and probably the 12.0 ones too.

**Comment by bmwiedemann** [comment]
updated

**Comment by toabctl** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**